include/flatbuffers/flatbuffers.h

```diff
@@ -288,7 +288,7 @@ struct VectorIterator : public
   }
 
   VectorIterator operator++(int) {
-    VectorIterator temp(data_);
+    VectorIterator temp(data_,0);
```

Comment:
Simplify this as suggested in https://github.com/google/flatbuffers/issues/3900 ?